provider/source/src/calendar/hijri.rs

```diff
@@ -32,7 +32,74 @@ impl crate::IterableDataProviderCached<CalendarHijriObservationalMeccaV1> for So
 impl DataProvider<CalendarHijriUmmalquraV1> for crate::SourceDataProvider {
     fn load(&self, req: DataRequest) -> Result<DataResponse<CalendarHijriUmmalquraV1>, DataError> {
         self.check_req::<CalendarHijriUmmalquraV1>(req)?;
-        let cache = HijriUmmAlQura::new_always_calculating().build_cache(1317..1567);
+
+        // From https://github.com/unicode-org/icu/blob/1bf6bf774dbc8c6c2051963a81100ea1114b497f/icu4c/source/i18n/islamcal.cpp#L87
+        const ICU4C_ENCODED_MONTH_LENGTHS: [u16; 1601 - 1300] = [
```

Comment:
question (nb): is this data more compact than what we do? Should we consider switching to this format?

Reply:
it's less compact